cli/src/test/resources/ide-projects/jmc/downloadMockLocation/org.openjdk.jmc-8.3.0-linux.gtk.x86_64/InstallTest.txt

```diff
@@ -0,0 +1 @@
+This is a test file.
```

Comment:
IMHO something is wrong here. Where does the `org.openjdk.` prefix come from?
Where does `linux.gtk.x86_64` come from?
We should build the unique location from these properties:
* tool
* edition
* version
* os (optional)
* arch (optional)
Further, I would recommend to not use any OS differences for tests unless really needed (if we have OS dependent code in the commandlet that we need to test).
Also I would suggest to not combine all the attributes into a single folder name.
We already have the folder based structure in `urls` so why not using the same here?
So use path `jmc/jmc/8.3.0/` instead of `org.openjdk.jmc-8.3.0-linux.gtk.x86_64`